packages/theme/src/index.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * material-ui theme for use with Backstage.
+ * {@link https://mui.com/customization/theming/ | material-ui theme} for use with Backstage.
```

Comment:
A lil bit of a risky link, since this links to the v5 documentation but we're still on v4. Could link to v4 perhaps? Would need to make sure we update whenever we move to v5 though

Reply:
Good call out. To remove maintenance, just opted to make it generic a link MUI but not the theme page, specifically. User can click a few links deep to get it pretty easily?